src/components/Lista/ItemList.jsx

```diff
@@ -102,8 +84,23 @@ const ItemList = ({ fullList, setFullList, handleDelete, toggleComplete, allProd
 
```

Comment:
komentuję tutaj, bo nie mogę powyżej. chodzi o to że lepiej przenieść wszystkie funkcje które robią fetch do zewnętrzengo pliku (np api.js)